pkg/smartcontract/manifest/manifest.go

```diff
@@ -35,6 +35,8 @@ type ABI struct {
 
 // Manifest represens contract metadata.
 type Manifest struct {
+	// Name is a contract's name.
+	Name string `json:"name"`
```

Comment:
We need to add names to our examples.

Reply:
This is specified during compilation in config.

Reply:
Yes, and configuration files for our examples do not contain names at the current moment. We can add them in case if someone will need it.

Reply:
And we should, it's an important change in contract's metadata, so examples need to be updated.

Reply:
Oh, I have missed config files are present in examples.